scripts/curate_basel.py

```diff
@@ -0,0 +1,127 @@
+import os
+import shutil
+import json
+import argparse
+
+root_data = '/Users/alex/data/basel-mp2rage_raw'
+output_data = '/Users/alex/data/basel-mp2rage_raw_curated'
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset Delbono Wakehealth to BIDS')
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    parser.add_argument("-o", "--outputdata",
+                        help="Path to output folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def main(root_data, output_data):
+    if os.path.isdir(output_data):
+        shutil.rmtree(output_data)
+    os.makedirs(output_data)
+
+    dict_images = {
+        "MP2RAGE_UNI_Images.nii.gz": "_UNIT1.nii.gz"
+    }
+
+    dict_der = {
+        "lesion_mask_sc.nii.gz": "_UNIT1_lesion-manual.nii.gz"
+    }
+
+    for dirs, subdirs, files in os.walk(root_data):
+        for file in files:
+            if file.endswith('.nii.gz') and file in dict_images.keys() or file in dict_der.keys():
```

Comment:
`.keys()` is redundant here

```suggestion
            if file.endswith('.nii.gz') and file in dict_images or file in dict_der:
```

Reply:
Also come to think of it, is this missing parens? As written it means

```suggestion
            if (file.endswith('.nii.gz') and file in dict_images) or file in dict_der:
```

but I think you meant
```suggestion
            if file.endswith('.nii.gz') and (file in dict_images or file in dict_der):
```